ibm_catalog.json

```diff
@@ -0,0 +1,92 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-icd-elasticsearch",
+        "label": "Elasticsearch",
+        "product_kind": "secure",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "reference_architecture",
+          "solution"
+        ],
+        "keywords": [
+          "elasticsearch",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "elasticsearch secure",
+          "database",
+          "nosql"
+        ],
+        "short_description": "Creates and configures an IBM Elasticsearch instance.",
+        "long_description": "This architecture supports creating and configuring an IBM Elasticsearch instance with kms encryption, cbr rules and autoscaling.",
```

Comment:
kms -> KMS
cbr -> CBR
Autoscaling -- we can remove I think.

Reply:
done